lib/interface_quda.cpp

```diff
@@ -52,7 +52,8 @@
 #define MAX(a,b) ((a)>(b)? (a):(b))
 #define TDIFF(a,b) (b.tv_sec - a.tv_sec + 0.000001*(b.tv_usec - a.tv_usec))
 
-#define spinorSiteSize 24 // real numbers per spinor
+// Is this even used?
```

Comment:
Not use, and should be deleted.  This is a remnant I missed from QUDA 0.1 😄

Reply:
Deleted